flight_club/sessions/views.py

```diff
@@ -19,65 +19,253 @@
 from flight_club.models.models import Beer, Session
 from flight_club.auth.views import login_required
 from flight_club.sessions.fc_sessions import FCSession
+from werkzeug.exceptions import BadRequestKeyError
 
+import datetime
 import flight_club.models.db_func as db_func
 
 bp = Blueprint("sessions", __name__, url_prefix="/sessions")
 
 
+def session_input_validator(session_id, date, host):
+    """This function is for validation the inputs for the session portion
+    of add session. It will type check and value check.
+
+    Args:
+        session_id : the id of the session
+        date : the date of the session
+        host : the host of the
+
+    Returns:
+        boolean indicating if checks failed
```

Comment:
To simplify this I'd suggest returning a single object with these as fields.  Specifically a [dataclass](https://docs.python.org/3/library/dataclasses.html)

Reply:
Hmm I've never used dataclass before, but I agree that would be a lot cleaner.